src/fixed.ts

```diff
@@ -61,6 +65,8 @@ export default class FixedThreadPool<Data = any, Response = any> {
     if (!isMainThread) {
       throw new Error('Cannot start a thread pool from a worker thread !!!')
     }
+    // TODO christopher 2021-02-07: Improve this check e.g. with a pattern or blank check
+    // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
```

Comment:
I think you can remove that rule disablement now that it's done at the file level.